tests/Shared/RepoTesting/Aspire.RepoTesting.targets

```diff
@@ -33,7 +33,7 @@
     `AspireProjectOrPackageReference` - maps to projects in `src/` or `src/Components/`
   -->
 
-  <Import Project="Sdk.props" Sdk="Aspire.AppHost.Sdk" Version="$(PackageVersion)" Condition="'$(IsAspireHost)' == 'true' and '$(RepoRoot)' == '' and '$(TestsRunningOutsideOfRepo)' == 'true'" />
+  <Import Project="Sdk.props" Sdk="Aspire.AppHost.Sdk" Version="9.3.0" Condition="'$(IsAspireHost)' == 'true' and '$(RepoRoot)' == '' and '$(TestsRunningOutsideOfRepo)' == 'true'" />
```

Comment:
Do we do this hardcoding every time?

Reply:
I'd love for that not to be the case but we haven't made the work to fix it. Main issue is that PackageVersion can't be used as that is a variable that changes depending on the project you are building, so when you are emitting a reference (like in this case or in the Directory.Packages.Helix.props) to a different package getting built, you don't want to use `PackageVersion` (as that is the package version of the project getting built) and instead you want to have a way to call a target into that reference to know whether or not that specific reference should be stable or not.

This is not super trivial and we haven't been able to prioritize it, so for now, we have to hard code each time we stabilize, but I'd love to not have to do this in the future :)